### DISCUSSION TITLE
Al final se va a aprender algun framework?

### DISCUSSION THREAD

**Discussion by VictorFalconi** [Launch-X-Latam/MisionFrontEnd#347] [Q&A]
Vi que el mission commander nos dice que se acabo la mision frontend. Que paso con React, Vue o Angular?

**Comment by doguedogue** [comment]
Fueron las bases de front end, por supuesto se puede profundizar con cualquiera de esos frameworks que mencionas

**Reply by VictorFalconi** [reply]
No pero yo pense que algun framework estaria incluido en el curso 😱
